backend/lucene/src/main/java/org/hibernate/search/backend/lucene/LuceneBackend.java

```diff
@@ -28,4 +31,19 @@ public interface LuceneBackend extends Backend {
 	 */
 	Optional<? extends Analyzer> normalizer(String name);
 
+	/**
+	 * Looks up the built-in and configured analyzers available to this backend.
+	 *
+	 * @param name The name of the analyzer.
+	 * @return An {@link Optional#empty() empty optional} if there is no analyzer configured with the given name.
+	 */
+	@Incubating
+	Optional<LuceneAnalyzerDescriptor> analyzerDescriptor(String name);
+
+	/**
+	 * @return A collection of built-in and configured analyzer descriptors available to this backend.
+	 */
+	@Incubating
+	Collection<LuceneAnalyzerDescriptor> analyzerDescriptors();
+
```

Comment:
IMO this is not really necessary as `IndexDescriptor` + the current `analyzer(...)` methods already expose all one could need.

But if you feel strongly about it... :shrug: